backend/tests/conftest.py

```diff
@@ -185,7 +185,7 @@ def _test_pushes(request):
     with responses.RequestsMock(assert_all_requests_are_fired=False) as resps:
         resps.add_callback(
             responses.GET,
-            re.compile("https://hg.mozilla.org/(.+)/json-rev/(.+)"),
+            re.compile("https://hg.mozilla.org/(.+)/json-automationrelevance/(.+)"),
             callback=_test_rev,
```

Comment:
`_test_rev` is the function that builds the response, you should update it too.
It is currently returning a response in the format json-rev returns, instead it should return a response in the same format as json-automationrelevance

Reply:
I have already made changes in `_test_rev` function
https://github.com/mozilla/code-coverage/pull/927/files/ab47697240fc5215f52bac28294404bf09e1f7b5#diff-aacea3e16f9fecd307f1a11567ba7a1abb8a6007266aec3b89dd65454130b76cR147
What other changes need to be done?

Reply:
The response returned by `_test_rev` should be in the json-automationrelevance format (e.g. https://hg.mozilla.org/mozilla-central/json-automationrelevance/0058e174c5d0a4f751398b5aefcce42adc511535).
You are still returning the response in the json-rev format: `resp = {"pushid": int(revision[:3]), "date": [time.time(), 0]}`.

Reply:
Changes pushed 7276fed
Are there any more fields to be added in response other than `pushid` and `date`?

Reply:
No, those should be enough